test/apps/uct_info/check_tls.sh.in

```diff
@@ -1,10 +1,12 @@
 #!/bin/sh
 
+# check_tls.sh [extra-tls-to-check]
```

Comment:
can we move this script to buildlib/, and make it .sh instead of .in.sh? since now it does not depend on m4 substitution

Reply:
> can we move this script to buildlib/, and make it .sh instead of .in.sh? since now it does not depend on m4 substitution

done